glob.go

```diff
@@ -80,14 +83,21 @@ func toNixPath(path string) string {
 
 // Glob returns all files that match the given pattern in the current directory.
 func Glob(pattern string, opts ...OptFunc) ([]string, error) {
+	prefix := "./"
+	if strings.HasPrefix(pattern, stringSeparator) {
```

Comment:
I'm not entirely sure about this part yet... DirFS won't allow to start on `/`, so to keep the API I'm doing this little hack here... but maybe we can just declare it a breaking change?

Reply:
What do you mean by that? `prefix = stringSeparator` and `WithFS(os.DirFS(prefix))` seems like you are doing exactly that.

As far as I understand the docs, `os.DirFS` has no concept of a current working directy. So normally I would expect the fs to be `os.DirFS("/")` so I can access everything. But if I open `file` I would expect it to open `/cwd/file` where `/cwd` is my current working directory. But instead, `os.DirFS("/") opens `/file`. So what we would need to keep track of is the current working directory right?

Reply:
fs.Open only accepts relative dirs

but I think this is indeed outside of the API. 

Before it was also only working on paths relative to current, and didn't accept absolute paths anyway (`/etc/foo/bar.conf` for example)